lib/platform/gitlab/index.ts

```diff
@@ -445,24 +464,32 @@ export async function getPr(iid: number): Promise<Pr> {
 
 export async function updatePr({
   number: iid,
-  prTitle: title,
+  prTitle,
   prBody: description,
   state,
 }: UpdatePrConfig): Promise<void> {
+  const url = `projects/${config.repository}/merge_requests/${iid}`;
+  const currentTitle = (await gitlabApi.getJson<{ title: string }>(url)).body
+    .title;
+  let title = prTitle;
+  // detect either draft prefix in case GitLab was recently upgraded
```

Comment:
We have a legacy feature where you can rename a PR to start with `rebase!` and that will trigger a rebase. It would conflict with this detection so either we need increased logic or a disclaimer about its compatibility.

Reply:
IMO the disclaimer is the better option, given that if a user adds `rebase!` to the beginning of the title, GitLab will stop treating it as a draft anyway. Do you want that in the docs about draftPR or somewhere else?

Reply:
Good point. In that case the docs for draftPr would be best.